x-pack/plugins/security_solution/public/common/components/event_details/cti_details/threat_details_view.tsx

```diff
@@ -125,31 +107,108 @@ const buildThreatDetailsItems = (enrichment: CtiEnrichment) =>
       };
     });
 
+const renderAccordion = (
```

Comment:
If the output of a function is JSX, we should type it as a Functional Component and use it as such.

Reply:
updated